src/test/scala/uk.gov.nationalarchives.fileformat/LambdaTest.scala

```diff
@@ -0,0 +1,68 @@
+package uk.gov.nationalarchives.fileformat
+
+import java.util.UUID
+
+import graphql.codegen.types.FFIDMetadataInput
+import io.circe.parser.decode
+import org.scalatest.matchers.should.Matchers.{equal, _}
+import uk.gov.nationalarchives.fileformat.AWSUtils._
+
+import scala.util.Try
```

Comment:
`import scala.util.Try` appears to be unnecessary import

Reply:
I've removed this one as well.